app/authenticators/jam-jwt.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
```

Comment:
@samanehsan would you be willing to factor out some of the copy/pasted code into exp-addons?